lib/Sema/CSApply.cpp

```diff
@@ -1233,11 +1233,32 @@ namespace {
                          ConstraintLocatorBuilder locator,
                          bool isImplicit, AccessSemantics semantics) {
       // Determine the declaration selected for this subscript operation.
-      auto selected = getOverloadChoice(
+      auto selected = getOverloadChoiceIfAvailable(
```

Comment:
They might be, I saw couple of valid ones too, do you want to be to double-check and maybe create separate PR to address any issues I find?